src/r_draw.c

```diff
@@ -530,7 +530,7 @@ static void R_DrawFuzzColumn_block(void)
     return;
 
   // [FG] draw only even pixels
-  dc_yl &= (int)~1;
+  dc_yl = (dc_yl + 1) & ~1;
   dc_yh &= (int)~1;
```

Comment:
This is good enough for me and is a one-liner.

Reply:
Maybe not not so great:
![woof0001](https://github.com/fabiangreffrath/woof/assets/56656010/116bbaaf-0b95-4a8c-9dd2-f268a10af176)

I have another solution but the changes will be much greater. It uses a `R_DrawVisSprite_fuzzblock()` function to avoid branching.